webview/platforms/qt.py

```diff
@@ -593,6 +593,38 @@ def _get_debug_port():
     def on_create_window(func):
         func()
 
+class Process(object):
+    exec_ = None
+    name = 'fake'
+    alive = True
+    daemon = False
+    pid = None
+    exitcode = None
+    authkey = b''
+    sentinel = None
+    
+    def join(self, timeout=None):
+        if timeout:
+            from time import sleep
+            sleep(timeout)
+            self.kill()
+        else:
+            self.exec_()
```

Comment:
if `join` called without `timeout` arg it starts gui loop

Reply:
Extract this class to `process.py`. All the platforms should return the same object. join() is the only needed function for now. If timeout is needed kill can be left as well.

Reply:
are you sure?
developers want other methods and properties(specially `close`)
other maybe other methods and properties wanted by someone.
I want `close` method